wrapper/src/main/java/software/amazon/jdbc/dialect/RdsMultiAzDbClusterMysqlDialect.java

```diff
@@ -44,8 +48,18 @@ public boolean isDialect(final Connection connection) {
     ResultSet rs = null;
     try {
       stmt = connection.createStatement();
-      rs = stmt.executeQuery(TOPOLOGY_QUERY);
-      return rs.next();
+      rs = stmt.executeQuery(TOPOLOGY_FUNC_EXIST_QUERY);
+
+      if (rs.next()) {
```

Comment:
NIT: is there any difference in doing
```
if (!rs.next()) {
  return false;
}
rs.close(); stmt.close();
execute query
etc...
```

Reply:
No, I don't think there's any difference.